### PR TITLE
Update compilation stages: move description from slides to source code

### PR DESCRIPTION
**Pull request by aelizaro**
- added preprocessor behavior description and command line
- added a note to show asm generation on compile stage

### CODE REVIEW

**Comment by aelizaro** [comment]
Do we want also to delete the presentation or keep it?

**Comment by ivankochin** [comment]
> Do we want also to delete the presentation or keep it?

I think that we want to delete the presentation.

File: 00_compilation_stages/00_00_preprocessor/00_00_00_simple/00_00_00_simple.cpp [hunk 1]
```diff
@@ -1,4 +1,16 @@
...
+// * Replaces #define macros with their values
+// * Selects the necessary pieces of code according to the conditions of #if,
+//   #ifdef and #ifndef
+// * Includes header files content
+//
+// Command example to run only preprocessor:
+// g++ -E 00_00_00_simple.cpp -o 00_00_00_simple.ii
+//
+// * -E runs preprocessor for a given file without compilation
+// * -o sets the name of the resulting file (extension name (.ii) is arbitrary)
```
Comment (ivankochin) [inline 00_compilation_stages/00_00_preprocessor/00_00_00_simple/00_00_00_simple.cpp]:
```suggestion
// * -o sets the name of the resulting file
```

File: 00_compilation_stages/00_00_preprocessor/00_00_00_simple/00_00_00_simple.cpp [hunk 2]
```diff
@@ -1,4 +1,16 @@
-// Show work of preprocessor(use -E)
+// Preprocessor behavior:
+// * Removes comments
+// * Replaces #define macros with their values
+// * Selects the necessary pieces of code according to the conditions of #if,
+//   #ifdef and #ifndef
+// * Includes header files content
+//
+// Command example to run only preprocessor:
```
Comment (ivankochin) [inline 00_compilation_stages/00_00_preprocessor/00_00_00_simple/00_00_00_simple.cpp]:
```suggestion
// Command example to run preprocessor only:
```

File: 00_compilation_stages/00_00_preprocessor/00_00_00_simple/00_00_00_simple.cpp [hunk 3]
```diff
@@ -1,4 +1,16 @@
-// Show work of preprocessor(use -E)
+// Preprocessor behavior:
+// * Removes comments
+// * Replaces #define macros with their values
+// * Selects the necessary pieces of code according to the conditions of #if,
+//   #ifdef and #ifndef
```
Comment (ivankochin) [inline 00_compilation_stages/00_00_preprocessor/00_00_00_simple/00_00_00_simple.cpp]:
```suggestion
// * Removes the code under `false` condition for #if, #ifdef and #ifndef
```

**Pull request merged**